purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -788,9 +791,26 @@ class Purchases internal constructor(
     }
     // endregion
 
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    fun redeemWebPurchase(webRedemptionLink: DeepLink.WebRedemptionLink, listener: RedeemWebResultListener) {
+        purchasesOrchestrator.redeemWebPurchase(webRedemptionLink, listener)
+    }
+
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    sealed class DeepLink {
+        class WebRedemptionLink internal constructor(internal val redemptionToken: String) : DeepLink()
```

Comment:
Naming suggestion: `WebPurchaseRedemption`. Also focussed on getting the word "purchase" in there, and I think `Link` can be omitted because it is already nested inside `DeepLink`.